src/screens/Principal/components/Blocks/components/Block/index.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './index.module.scss';
+
+function Block({ uuid, number, certified, color, fee, onDragEnd, onDragStart, bigBlock }) {
+  const colorDark = `${color}Dark`;
+  const MAX_FEE = 1;
+  const MAX_POINTS = 7;
+  const points = (fee / MAX_FEE) * MAX_POINTS + 1;
+  const pointsClass = [];
+
+  for (let i = 1; i <= points; i++) {
+    pointsClass.push(`point${i}`);
+  }
+
+  return (
+    <div
+      id={uuid}
+      className={` ${bigBlock ? styles.containerBigBlock : styles.containerSmallBlock} ${
+        styles[colorDark]
+      } containerBlockDrag`}
+      draggable
+    onDragEnd={onDragEnd}
+    onDragStart={onDragStart}
+     >
```

Comment:
Por lo general se mueven las props que sean explicitamente true debajo de las dems y de paso podés revisar la identación?:
```suggestion
    <div
      id={uuid}
      className={` ${bigBlock ? styles.containerBigBlock : styles.containerSmallBlock} ${
        styles[colorDark]
      } containerBlockDrag`}
      onDragEnd={onDragEnd}
      onDragStart={onDragStart}
       draggable
     >
```